library/manageiq_provider.py

```diff
@@ -150,11 +151,15 @@ class ManageIQ(object):
     """
 
     OPENSHIFT_DEFAULT_PORT = '8443'
-    provider_types = {
+
+    PROVIDER_TYPES = {
         'openshift-origin': 'ManageIQ::Providers::Openshift::ContainerManager',
         'openshift-enterprise': 'ManageIQ::Providers::OpenshiftEnterprise::ContainerManager',
         'amazon': 'ManageIQ::Providers::Amazon::CloudManager'}
```

Comment:
Not yet supported, but definitely a good addition.

@yaacov We welcome contributions, especially on holidays :)